src/Workspaces/CSharp/Portable/CodeGeneration/CSharpSyntaxGenerator.cs

```diff
@@ -2680,6 +2680,8 @@ internal static BaseParameterListSyntax GetParameterList(SyntaxNode declaration)
                     return ((ParenthesizedLambdaExpressionSyntax)declaration).ParameterList;
                 case SyntaxKind.SimpleLambdaExpression:
                     return SyntaxFactory.ParameterList(SyntaxFactory.SingletonSeparatedList(((SimpleLambdaExpressionSyntax)declaration).Parameter));
+                case SyntaxKind.LocalFunctionStatement:
+                    return ((LocalFunctionStatementSyntax)declaration).ParameterList;
```

Comment:
Can you fix `WithParameterList` below as well for symmetry?

Reply:
`SyntaxKind.LocalFunctionStatement` is already supported in `WithParameterList`: https://github.com/dotnet/roslyn/blob/b966e6700749381dcc9a8b73450214aa376e74eb/src/Workspaces/CSharp/Portable/CodeGeneration/CSharpSyntaxGenerator.cs#L2690-L2709

Reply:
Ok thanks